src/js/worklets/chat/messages.js

```diff
@@ -1,4 +1,4 @@
-import { useDerivedValue, withTiming, interpolate, useAnimatedScrollHandler, runOnJS } from 'react-native-reanimated';
+import { useDerivedValue, withTiming, interpolate, useAnimatedScrollHandler } from 'react-native-reanimated';
```

Comment:
this file is getting quite big, will refactor in next PR and will break in separate files (navigation, composer etc.)